lib/glific_web/flows/flow_editor_controller.ex

```diff
@@ -226,6 +229,34 @@ defmodule GlificWeb.Flows.FlowEditorController do
     json(conn, %{results: results})
   end
 
+  @doc """
+  Fetching single interactive template and returning in format that is understood by flow editor
+  or
+  Return error Interactive message not found
+  """
+  @spec interactive_template(Plug.Conn.t(), nil | maybe_improper_list | map) :: Plug.Conn.t()
+  def interactive_template(conn, params) do
+    [id] = params["vars"]
+    {:ok, id} = Glific.parse_maybe_integer(id)
+
+    case Repo.fetch_by(InteractiveTemplate, %{id: id}) do
```

Comment:
For the consistency can we please use a function call get_interactive_template!(id)

Reply:
@pankaj-ag `get_interactive_template!(id) ` raises `Ecto.NoResultsError` if the Interactive Template does not exist. Therefore we nee to use i `try and rescue` clause to return proper error.
Using `Repo.fetch_by` returns `{:ok, _}` and `{:error, _}`

Reply:
I will clean this later.